state/txindex/kv/kv.go

```diff
@@ -52,7 +53,7 @@ func (txi *TxIndex) Get(hash []byte) (*abci.TxResult, error) {
 
 	rawBytes, err := txi.store.Get(txi.ctx, ds.NewKey(hex.EncodeToString(hash)))
 	if err != nil {
-		if err == ds.ErrNotFound {
+		if errors.Is(err, ds.ErrNotFound) {
```

Comment:
consider wrapping error below in line 68:

```
	if err != nil {
		return nil, fmt.Errorf("error reading TxResult: %v", err)
	}
```

Reply:
thank you for feedback, i think still use this refactor, if change maybe break the previous issues -> https://github.com/rollkit/rollkit/pull/1585